aptos-move/block-executor/src/executor.rs

```diff
@@ -393,63 +269,68 @@ where
             // Iterate round robin over workers to do commit_hook.
             *worker_idx = (*worker_idx + 1) % post_commit_txs.len();
 
+            if let Some(fee_statement) = last_input_output.fee_statement(txn_idx) {
```

Comment:
if we have the dedicated function `block_truncated_at_idx`, this one should just not return an Option

Reply:
i'm probably missing smt obvious, but this matches Success and SkipRest, and truncated at index matches SkipRest and Abort?